_glaredb/sq-functions/parquet_scan.md

```diff
@@ -6,37 +6,163 @@ parent: SQL functions
 
 # `parquet_scan`
 
-Read a parquet file. The file may exist locally or remotely.
+Read parquet files from a local filesystem or supported object stores.
+
+URLs provided to `parquet_scan` may also include globbing characters to allow
+scanning multiple files when reading from S3, GCS, or a local filesystem.
+Globbing is not allowed when when querying parquet files over HTTP/HTTPS.
 
 ## Syntax
 
 ```sql
+-- Single url or path.
 parquet_scan(<url>)
+-- Multiple urls or paths.
+parquet_scan([<url>])
```

Comment:
could this be:

```sql
parquet_scan(<url>[, ...])
```
?

Reply:
No, since we require the opening and closing brace which that doesn't indicate. I opted to be a bit more explicit with these syntax definitions since we use `[...]` to indicate optional elsewhere.